MHVLib/Methods/MHVMethod.h

```diff
@@ -38,6 +46,11 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, assign) int version;
 
 /**
+ A MHVMethodType indicating the method type
+ */
+@property (nonatomic, assign, readonly) MHVMethodType type;
+
+/**
```

Comment:
I'm don't think these are methods at all, and should not be subclassed or added to the MHVMethod class. There is a list of methods on the developer site (https://developer.healthvault.com/Methods), we should avoid adding things to this class that are not methods listed in the dev portal.